lib/api/controllers/adminController.js

```diff
@@ -52,6 +55,35 @@ class AdminController extends BaseController {
     ]);
   }
 
+  decryptSecrets (request) {
+    assertHasBody(request);
+    assertBodyHasAttribute(request, 'secrets');
+    assertBodyHasAttribute(request, 'vaultKey');
+
+    const vaultKey = request.input.body.vaultKey,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use destructuring syntax for these assignments from "request.input.body". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS3514)